Bottom: d41a8ff53cb7c03079fd160af9d4be9dbee9d241
Top:    efe3c8ad24aac2c8efa16e23395752e21dfc1eac
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 23:40:00 +0100

Refresh of 3d-mouse-cursor-and-other-proj

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 2ee066c..fe2c11a 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1497,7 +1497,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
      world vector along +ve Z axis, (0, 0, 1). */
   /* FIXME: This isn't strictly correct, as I've ignored the matrix
             elements for homogeneous coordinates. */
-    /* NB: last_modelview_matrix is transposed in memory! */
+  /* NB: last_modelview_matrix is transposed in memory! */
   reverse_layers = last_modelview_matrix[2][2] < 0;
   Settings.ShowSolderSide = reverse_layers ? !Settings.ShowSolderSide : Settings.ShowSolderSide;
